mesheryctl/internal/cli/root/system/fixtures/.meshery/config.yaml

```diff
@@ -0,0 +1,13 @@
+contexts:
+  local:
+    endpoint: http://localhost:9081
+    token: Default
+    platform: docker
+    adapters:
+    - meshery-istio
+    channel: stable
+    version: v0.5.10
```

Comment:
Should this be `latest` or is this sample config simply a starting point and something that each integration test might update with different values?